src/repositories/following.repository.ts

```diff
@@ -0,0 +1,26 @@
+import { FollowingModel } from "./../models/Following.model";
+
+export const getList = async (queryParams: object) => {
+  const query = {};
+  const options = {
+    ...queryParams,
+  };
+  const result = await FollowingModel.paginate(query, options);
+  return result;
+};
```

Comment:
```suggestion
export const getList = async (query: object, options: object) => {
  return FollowingModel.paginate(query, options);
};
```
check tweet repositories